lib/root_loader.rb

```diff
@@ -23,19 +25,45 @@ def load account, filename
         end
 
         root_policy_resource = Loader::Types.find_or_create_root_policy(account)
+        policy_version = policy_version(role: admin, policy: root_policy_resource, filename: filename)
```

Comment:
split long line:
```ruby
policy_version = policy_version(
  role: admin,
  policy: root_policy_resource,
  filename: filename  
)
```
and per other comment, use something different from local var name than the method name, `policy_ver` or something.